src/ontology/mondo-edit.obo

```diff
@@ -159159,6 +159159,7 @@ xref: SCTID:4170004 {source="MONDO:equivalentTo"}
 xref: UMLS:C4551623 {source="MEDGEN:1645042", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0020066 {source="DC-OMIM:130060", source="DOID:0080727", source="MESH:C562625", source="NCIT:C125701", source="OMIM:130060", source="Orphanet:1899"} ! Ehlers-Danlos syndrome
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0007525 {source="MONDO:CLINGEN"}
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2197 {source="OMIM:130060"} ! COL1A1
```

Comment:
Looks good https://omim.org/entry/130060

Reply:
@twhetzel this is not correct. 
OMIM:130060 is not equivalent to this term. 
Note that OMIM:130060 not being equivalent anymore might have happened after the data release. However, this equivalence is not showing in this PR either.